qiskit/algorithms/algorithm_result.py

```diff
@@ -31,7 +31,6 @@ def __str__(self) -> str:
                 and not inspect.isfunction(value)
                 and hasattr(self, name)
             ):
-
```

Comment:
It seems something went wrong in the formatting of the code, maybe a mismatch in the Black version? The CI doesn't seem to complain but it would probably be good to reduce the changes to the logical changes still 🙂

Reply:
Yeah, I think I ran `black` not `tox -eblack` so it picked up extra files and formatted them.  I reverted this to get rid of the extra edits